.gitattributes

```diff
@@ -1,2 +1,2 @@
-* text=auto eol=lf
 *.bat text eol=crlf
+* text=auto eol=lf
```

Comment:
This seems like an unnecessary change... I guess the idea is to have the lines sorted alphabetically?

Reply:
* is greedy, so .bat files were just matching the first line and were taking LF as EOL.
I became aware of it when upgrading gradle, it wrote a new gradelw.bat with the wrong EOL.

Reply:
TIL! Thanks for commenting.